src/dsql/parse.y

```diff
@@ -695,6 +695,7 @@ using namespace Firebird;
 %token <metaNamePtr> TIMEZONE_NAME
 %token <metaNamePtr> UNICODE_CHAR
 %token <metaNamePtr> UNICODE_VAL
+%token <metaNamePtr> FORMAT
```

Comment:
Please move it before `LOCKED`, i.e., alphabetical order.

Reply:
Sorry, instead, rebase it with master and put in section `// tokens added for Firebird 6.0`.

Reply:
Section is there already :), move it.